doc: Add known issue for KRKNWK-20035 (#20718)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-9-0 v2-8-0

> KRKNWK-20035: Door Lock Attributes does not persists after reboot/reset

Reviewer: ```suggestion
KRKNWK-20035: Door Lock Attributes do not persist after reboot or reset
```

>   Persistent storage fails to load due to wrong value of :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE` Kconfig option.

Reviewer: ```suggestion
  The persistent storage fails to load due to a wrong value in the :kconfig:option:`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE` Kconfig option.
```

>   **Workaround:** Cherry-pick changes from PR #20713  in the `sdk-nrf`_ repository.

Reviewer: ```suggestion
  **Workaround:** Cherry-pick changes from PR #20713 in the `sdk-nrf`_ repository.
```
